src/Module/Settings/Profile/Photo/Crop.php

```diff
@@ -185,9 +182,6 @@ protected function content(array $request = []): string
 		}
 
 		$Image = Photo::getImageForPhoto($photos[0]);
-		if (empty($Image)) {
-			throw new HTTPException\InternalServerErrorException();
-		}
```

Comment:
Please check for `$Image->isValid()` instead.

Reply:
Thanks, done.